srcpkgs/vorta/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'vorta'
+pkgname=vorta
+version=0.8.9
+revision=1
+build_style=python3-module
+#Running pytest as root leads 'test_export_fail_unwritable' to fail.
+#'test_ssh_dialog' fails under xbps-src (either 'xbps-src check' or
+#executing pytest into 'xbps-src chroot' enviroment) and I don't know why. 
+#On the other hand, it passes if pytest(ed) chroot(ed) into 
+#the 20221001_glibc release (either as root or user). May be an issue 
+#specific to xbps-src?
+make_check_args="--deselect=tests/test_import_export.py::test_export_fail_unwritable
+ --deselect=tests/test_repo.py::test_ssh_dialog"
+hostmakedepends="python3-setuptools python3-wheel python3-pip"
```

Comment:
Remove pip, it's only downloading dependencies behind your back and let's you ignore issues with the template.

Reply:
Without pip compilation breaks very soon. This is the error: `distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpoe4d9e4t', '--quiet', 'pip>=10']' returned non-zero exit status 1.`

Do we really need to remove pip? I was partially inspired by borg's template in which `hostmakedepends` include it.

Reply:
The borg template is wrong and should be fixed, feel free to open a PR.
Vorta has pip in setup_requires in setup.cfg which is wrong, it doesn't need pip. Could you patch it out and send a PR upstream?

Reply:
Sure. There will be a learning curve but nothing I couldn't deal with.

Reply:
Should I include this conversation l the PRs (to upstream and to borg's)?

Reply:
Since I have no technical arguments to support the PRs.

Reply:
Patch included in this PR.